qgepplugin/processing_provider/test_qgep_swmm.py

```diff
@@ -0,0 +1,18 @@
+from QgepSwmm import QgepSwmm
```

Comment:
What is this file exactly?
C:/ doesn't look very cross platform ...

Reply:
I use this file to run the qgep-swmm function outside from QGIS. I could also remove it if it is confusing.

Reply:
Should you not just use the variables for these paths (Line 7 - 10) as you have used in e.g.  qgepplugin/processing_provider/swmm_execute.py 
        inp_file = self.parameterAsFileOutput(parameters, self.INP_FILE, context)
        swmm_cli = os.path.abspath(ProcessingConfig.getSetting("SWMM_PATH"))
        
        And use the actual pg_service of the project you are working on instead of a hard wirede "pg_gep_gruner" ? Line 5